### PR TITLE
simplified output string for sentiment analysis

### PR DESCRIPTION
**Pull request by miiklay**
per feedback on https://github.com/capeprivacy/docs/pull/33

### CODE REVIEW

File: sentiment_analysis/app/app.py [hunk 1]
```diff
@@ -66,11 +66,6 @@ def cape_handler(text):
...
-        result,
-        " with a probability of ",
-        output_data[0][output_result] * 100,
-        "%.",
-    )
-    
\ No newline at end of file
+    
+    prob = output_data[0][output_result] * 100
+    return (str(float(f'{prob:.2f}')) + "% " + result) or "You've stumped me! Please try a different phrase."
```
Comment (bendecoste) [inline sentiment_analysis/app/app.py:71]: When would this `or` happen? Feels a little hard to parse what this is doing for a reference function 🤔

Comment (annieltan) [reply sentiment_analysis/app/app.py:71]: ☝️ agree with ben

also, for the string output, can you do `f'{prob:.2f}% {result}'` ?

Comment (ekloberdanz) [reply sentiment_analysis/app/app.py:71]: I don't think that the ```or``` is needed. Words that are not in the model's vocabulary are handled to be skipped, so that if there is an unknown word in a sentence, the model will simply assess all known words.

Comment (ekloberdanz) [reply sentiment_analysis/app/app.py:71]: Nevertheless, the ```or``` does not hurt. I cannot think of a situation when it will be executed, but better to have it just in case. Regarding the formatting, it looks good to me. I tested it and it renders nicely as follows:

78.08% positive

Hence, I approve the PR.

**Review by ekloberdanz — APPROVED** [review]

File: sentiment_analysis/app/app.py (hunk 1 above)
Comment (miiklay) [reply sentiment_analysis/app/app.py:71]: I defer to @ekloberdanz's approval and will leave the `or` since it's her function :)
